app/views/nsm/steps/view_claim/allowed_work_items.html.erb

```diff
@@ -8,88 +8,101 @@
   <%= govuk_details(summary_text: t('.check_totals')) do %>
     <%=
       head = [
-        { text: t('.item'), width: 'govuk-!-width-one-third' },
+        { text: '', width: 'govuk-!-width-one-third' },
         { text: t('.time'), numeric: true },
         { text: t('.net_cost'), numeric: true },
         { text: t('.allowed_time'), numeric: true },
         { text: t('.allowed_net_cost'), numeric: true },
       ]
+
       total_cost = BigDecimal('0')
       allowed_total_cost = BigDecimal('0')
       vat_rate = current_application.firm_office.vat_registered == 'true' ? Pricing.new(@claim).vat : 0
-      rows = @work_items.group_by(&:work_type).map do |work_type, work_items_for_type|
+      sorted_work_items = @work_items.sort_by(&:sort_position)
+
+      rows = sorted_work_items.group_by(&:work_type).map do |work_type, work_items_for_type|
         time = work_items_for_type.sum(&:time_spent)
         net_cost = work_items_for_type.sum(&:total_cost)
         allowed_time = work_items_for_type.sum { _1.allowed_time_spent || _1.time_spent }
         allowed_net_cost = work_items_for_type.sum(&:allowed_total_cost)
         total_cost += net_cost
         allowed_total_cost += allowed_net_cost
+
         [
           { text: t("summary.nsm/cost_summary/work_items.#{work_type.to_s}"), width: 'govuk-!-width-one-quarter' },
-          { text: format_period(time), numeric: true },
+          { text: format_period(time, style: :minimal_html), numeric: true },
           { text: NumberTo.pounds(net_cost), numeric: true },
-          { text: format_period(allowed_time), numeric: true },
+          { text: format_period(allowed_time, style: :minimal_html), numeric: true },
           { text: NumberTo.pounds(allowed_net_cost), numeric: true },
         ]
       end
 
       foot = [
         { text: t(".total"), width: 'govuk-!-width-one-quarter' },
         { text: '', numeric: true },
-        { text: NumberTo.pounds(total_cost), numeric: true },
+        { text: govuk_visually_hidden(t('.accessibility.net_cost')) + ' ' + NumberTo.pounds(total_cost), numeric: true },
```

Comment:
String concatenation isn't hugely idiomatic in ruby, and this will lead to unnecessarily unsafe strings. Consider:

```
safe_join([govuk_visually_hidden(t('.accessibility.net_cost')) , NumberTo.pounds(total_cost)])
```